src/pages/commits/CommitsChart.tsx

```diff
@@ -0,0 +1,139 @@
+import { useMemo } from "react";
+import styled from "@emotion/styled";
+import { useLocation } from "react-router-dom";
+import { Accordion } from "components/Accordion";
+import { size } from "constants/tokens";
+import { useUpdateURLQueryParams } from "hooks";
+import { ChartTypes, Commits, ChartToggleQueryParams } from "types/commits";
+import { queryString } from "utils";
+import { ChartToggle } from "./ActiveCommits/ChartToggle";
+import { Grid, SolidLine } from "./ActiveCommits/Grid";
+import { GridLabel } from "./ActiveCommits/GridLabel";
+import {
+  getAllTaskStatsGroupedByColor,
+  findMaxGroupedTaskStats,
+} from "./ActiveCommits/utils";
+import {
+  getCommitKey,
+  getCommitWidth,
+  RenderCommitsChart,
+} from "./RenderCommit";
+import { FlexRowContainer, CommitWrapper } from "./styles";
+
+const { parseQueryString, getString } = queryString;
+
+const DEFAULT_CHART_TYPE = ChartTypes.Absolute;
+const DEFAULT_OPEN_STATE = "True";
+
+interface Props {
+  versions?: Commits;
+  hasTaskFilter?: boolean;
+  hasError?: boolean;
+}
+
+export const CommitsChart: React.FC<Props> = ({
+  versions,
+  hasTaskFilter,
+  hasError = false,
+}) => {
+  const { search } = useLocation();
+  const updateQueryParams = useUpdateURLQueryParams();
+  const parsed = parseQueryString(search);
+
+  const onChangeChartType = (chartType: ChartTypes): void => {
+    updateQueryParams({
+      [ChartToggleQueryParams.chartType]: chartType,
+    });
+  };
+
+  const chartType =
+    (getString(parsed[ChartToggleQueryParams.chartType]) as ChartTypes) ||
+    DEFAULT_CHART_TYPE;
+
+  const onChangeChartOpen = (open: boolean): void => {
+    const chartOpen = open ? "True" : "False";
+    updateQueryParams({
+      [ChartToggleQueryParams.chartOpen]: chartOpen,
+    });
+  };
+
+  const chartOpen =
+    ((getString(parsed[ChartToggleQueryParams.chartOpen]) as ChartTypes) ||
+      DEFAULT_OPEN_STATE) === "True";
```

Comment:
```js
const chartOpen = Boolean(parsed[ChartToggleQueryParams.chartOpen] || true)
```

Reply:
The reason I checked equality to a string is because `Boolean("false")` evaluates to `true` 😔 I don't know if there's a way I can get around that

I can only think of something like this

```
  const chartOpen = parsed[ChartToggleQueryParams.chartOpen]
    ? parsed[ChartToggleQueryParams.chartOpen] === "true"
    : DEFAULT_OPEN_STATE;
```